x/authenticator/authenticator/interface.go

```diff
@@ -5,61 +5,60 @@ import (
 )
 
 const (
-	// SignatureVerificationAuthenticator is a Type of authenticator that specific deals with
-	// secp256k1 signatures,
+	// SignatureVerificationAuthenticatorType represents a type of authenticator specifically designed for
+	// secp256k1 signature verification.
 	SignatureVerificationAuthenticatorType = "SignatureVerificationAuthenticator"
```

Comment:
we should probably move this to `x/authenticator/authenticator/signature_authenticator.go`

Reply:
Done :+1: